src/components/FeatureChecking/withFeatureProvider.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import FeatureProvider from './FeatureProvider';
+import type { FeatureConfig } from './flowTypes';
+
+type WrapperProps = {
+    features: FeatureConfig,
+};
+
+function withFeatureProvider(WrappedComponent: React.ComponentType<any>) {
+    function WrapperComponent({ features, ...props }: WrapperProps) {
+        return (
+            <FeatureProvider features={features}>
+                <WrappedComponent {...props} />
```

Comment:
Does this forward refs as expected?

Reply:
fixed

Reply:
looks like the makeResponsive HOC doesn't though